openassessment/fileupload/views_filesystem.py

```diff
@@ -17,8 +17,6 @@ def filesystem_storage(request, key):
     """
     Uploading and download files to the local filesystem backend.
     """
-    if isinstance(key, unicode):
-        key = key.encode("utf-8")
```

Comment:
Hmm, I'm a bit worried about this.  What happens further downstream when S3 receives a key with non-ASCII characters?